setup.py

```diff
@@ -156,24 +156,24 @@
         ]
     },
     install_requires=[
-        'click==7.0',
-        'ipaddress==1.0.23',
-        'jsondiff==1.2.0',
-        'jsonpatch==1.32.0',
-        'm2crypto==0.31.0',
-        'natsort==6.2.1',  # 6.2.1 is the last version which supports Python 2. Can update once we no longer support Python 2
-        'netaddr==0.8.0',
-        'netifaces==0.10.7',
-        'pexpect==4.8.0',
-        'pyroute2==0.5.14',
-        'requests==2.25.0',
+        'click>=7.0',
+        'ipaddress>=1.0.23',
+        'jsondiff>=1.2.0',
+        'jsonpatch>=1.32.0',
+        'm2crypto>=0.31.0',
+        'natsort>=6.2.1',  # 6.2.1 is the last version which supports Python 2. Can update once we no longer support Python 2
```

Comment:
You can remove this comment; sonic-utilities is now Python 3-only. Could also try bumping it up to 7.1.1.

Reply:
Bumped and removed.